forward/forward_test.go

```diff
@@ -35,7 +35,8 @@ func TestCloneRequest(t *testing.T) {
 	const rawPath = "/%E4%B8%9C%E6%96%B9Project"
```

Comment:
Thanks @fffw!  I wish I understood why setting Opaque breaks ancestry.com, but I'm happy the fix works anyway.